test/ConverterTest.php

```diff
@@ -32,6 +41,13 @@ private static function resize10x10Command()
         return $command;
     }
 
+    private static function command($config)
```

Comment:
Please combine `resize10x10bgFF00FFcanvas20x20quality41Command()` and  `resize10x10Command()` functions to `command()`

Reply:
done